mockdate/index.d.ts

```diff
@@ -0,0 +1,28 @@
+// Type definitions for mockdate 2.0
+// Project: https://github.com/boblauer/MockDate
+// Definitions by: Bruno Leonardo Michels <https://github.com/brunolm>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+declare module "mockdate" {
+    export default MockDate;
```

Comment:
If I do `export as namespace MockDate;` it doesn't work on Node. Would it be ok to use `declare module "mockdate" {` and keep the rest as you said?